.devcontainer/README.md

```diff
@@ -0,0 +1,50 @@
+# Ubuntu 16.04 Remote Dev Container Instructions
+
+## Who's interested in this and why?
+* For those developers who'd like to validate their changes on Ubuntu 16.04, we provide a sandboxed development environment using the 'Remote Development Container' features of Microsoft's Visual Studio Code.
+* This feature pre-configures a dev environment with all the tools you'll need to build stellar-core
+
+
+## Pre-requisites: What do I need to use it?
+* Install [Visual Studio Code](https://code.visualstudio.com/Download) for your platform
+* Install the [Remote Development Extension Pack](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.vscode-remote-extensionpack)
+* Install [Docker](https://www.docker.com) for your platform
+  * If you are installing Docker on Windows, please make sure your settings have enabled Linux, rather than Windows containers.
+* Special note for Windows users:
+  * This feature mounts your git repository into the container so that you can use it from Linux.
```

Comment:
Windows: it seems that the git working folder is all borked because of line ending differences between platforms. Not sure what the best way around this (assuming the shared folder should be usable from both sides)